manifests/charts/gateway/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.defaultPodDisruptionBudget.enabled }}
+{{- if (semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion) }}
+apiVersion: policy/v1
```

Comment:
I would just only support v1 at this point. Its new feature (so no backwards compat needed) and support for 1.20 is going away in this release anyways